tests/test.cpp

```diff
@@ -129,7 +129,12 @@ static void testGeometryCollection() {
 }
 
 static void testFeature() {
+#if !defined(__GNUC__) || __GNUC__ >= 5
     feature<double> pf { point<double>() };
```

Comment:
👍 Let's avoid the macros by adding a constructor to `feature` with default arguments for `properties` and `id`.